camel-chimera-mapping-template/src/main/java/com/cefriel/rdf/MaptTemplateProcessor.java

```diff
@@ -148,36 +144,46 @@ else if ((params.resourceCustomFunctions() == null) && (params.customFunctions !
                 formatter = Util.createFormatter(params.formatterFormat());
             }
 
+            TemplateExecutor templateExecutor;
```

Comment:
Move TemplateExecutor as a field of the class MaptTemplateProducer to reuse the same instance across multiple executions of the same route